src/XIOPAK.H

```diff
@@ -95,7 +105,7 @@ int findFile1( const char* drvDir, const char* fName, char* fPathChecked=NULL);
 BOO findFile( const char* fName, const char* path, char* fPathFound);	// searches . first. NULL path searches env path.
 
 const char * FC xgetdir( void);
-SEC    FC xchdir( const char *, int erOp=WRN);
+SEC    FC xchdir( filesys::path, int erOp=WRN);
```

Comment:
This requires filesystem to be known externally.  Can't the old signature be retained (or a std::string variant) and suitably mapped to filesys::path within the functions?

Way back when, this code ran both on PCs and on a Data General minicomputer.  The idea of things like xiopak and envpak was to hide the OS difference -- exactly our problem here.